Sources/SQLKit/Builders/SQLDropTriggerBuilder.swift

```diff
@@ -0,0 +1,49 @@
+/// Builds `SQLDropTrigger` query
+///
+///   conn.drop()
+///
+/// See `SQLQueryBuilder` for more information.
+public final class SQLDropTriggerBuilder: SQLQueryBuilder {
+    /// `SQLDropTrigger` query being built.
+    public var dropTrigger: SQLDropTrigger
+
+    /// See `SQLQueryBuilder`.
+    public var database: SQLDatabase
+
+    /// See `SQLQueryBuilder`.
+    public var query: SQLExpression {
+        return self.dropTrigger
+    }
+
+    public init(_ dropTrigger: SQLDropTrigger, on database: SQLDatabase) {
+        self.dropTrigger = dropTrigger
+        self.database = database
+    }
+
+    /// The optional `IF EXISTS` clause suppresses the error that would normally
+    /// result if the table does not exist.
+    public func ifExists() -> Self {
+        dropTrigger.ifExists = true
+        return self 
+    }
+
+    /// The optional `CASCADE` clause drops other objects that depend on this type
+    /// (such as table columns, functions, and operators), and in turn all objects
+    /// that depend on those objects.
+    public func cascade() -> Self {
+        dropTrigger.cascade = true
+        return self
+    }
+}
+
+// MARK: Connection
+
+extension SQLDatabase {
```

Comment:
Nit: This extension should be at the top of the file. The reasoning being this is the method the user is probably looking for, so having it at the top makes it easy to find.